diff --git a/dist/node/chunks/dep-bb8a8339.js b/dist/node/chunks/dep-bb8a8339.js
index 417b6a1589395acefc8443e821a4960cf951e247..bc5806cf3e6646ba36e723df9420433c0f45c2d3 100644
--- a/dist/node/chunks/dep-bb8a8339.js
+++ b/dist/node/chunks/dep-bb8a8339.js
@@ -62392,7 +62392,7 @@ var requiresPort = function required(port, protocol) {
 (function (exports) {
 	var common   = exports,
 	    url      = require$$0$9,
-	    extend   = require$$0$6._extend,
+	    extend   = Object.assign,
 	    required = requiresPort;
 
 	var upgradeHeader = /(^|,)\s*upgrade\s*($|,)/i,
@@ -63761,7 +63761,7 @@ var wsIncoming = {
 
 (function (module) {
 	var httpProxy = module.exports,
-	    extend    = require$$0$6._extend,
+	    extend    = Object.assign,
 	    parse_url = require$$0$9.parse,
 	    EE3       = eventemitter3Exports,
 	    http      = require$$1$1,
